pyirf/interpolation.py

```diff
@@ -54,45 +55,118 @@ def interpolate_effective_area_per_energy_and_fov(
     aeff_interp = griddata(grid_points, effective_area, target_point, method=method).T
     # exp it and set to zero too low values
     aeff_interp = np.exp(aeff_interp)
-    aeff_interp[aeff_interp < min_effective_area * 1.1] = 0  # 1.1 to correct for numerical uncertainty and interpolation
-    return u.Quantity(aeff_interp, u.m**2, copy=False)
+    aeff_interp[
+        aeff_interp < min_effective_area * 1.1
+    ] = 0  # 1.1 to correct for numerical uncertainty and interpolation
+    return u.Quantity(aeff_interp, u.m ** 2, copy=False)
 
 
 def interpolate_energy_dispersion(
-    energy_dispersions,
-    grid_points,
-    target_point,
-    method='linear',
+    edges, hists, m, mprime, axis, quantile_resolution=1e-3
```

Comment:
hists should be `binned_pdf`, we don't expect a histogram here (as in event counts), correct?